node/lib/Logger.ts

```diff
@@ -20,62 +22,89 @@ export class Logger implements ILogger {
             this.globalLogLevel = logLevel;
         }
 
-        let consoleTransport = new winston.transports.Console({
+        this.consoleTransport = new winston.transports.Console({
             colorize: true,
             level: this.globalLogLevel,
         });
 
         this.logger.configure({
             transports: [
-                consoleTransport,
+                this.consoleTransport,
             ],
         });
 
-        this.transportList.push(consoleTransport);
+        this.transportList.push(this.consoleTransport);
 
-        if (Logger.flag === true && filename ) {
-            let fileTransport = new winston.transports.File({
+        if (Logger.flag === true && filename) {
```

Comment:
to prevent a transport being added twice, remember the error we saw the 1st time I demo-ed this, this was to fix that. Of course we have moved from .add to .configure, so we might not need that check anymore..

---
In reply to: [105340423](https://github.com/openT2T/opent2t/pull/38#discussion_r105340423) [](ancestors = 105340423)